roofit/roofitcore/inc/RooExtendPdf.h

```diff
@@ -23,8 +23,8 @@ class RooExtendPdf : public RooAbsPdf {
 public:
 
   RooExtendPdf() ;
-  RooExtendPdf(const char *name, const char *title, const RooAbsPdf& pdf, 
-	       const RooAbsReal& norm, const char* rangeName=0) ;
+  RooExtendPdf(const char *name, const char *title, RooAbsPdf& pdf,
+	       RooAbsReal& norm, const char* rangeName=0) ;
```

Comment:
here and below: this is not backward-compatible, you can't pass a `RooAbsPdf` rvalue to the constructor anymore. Is that ok?

Reply:
About the rvalue ...
That didn't exist when RooFit was designed. If you pass one, it will save the pointer, let the ravlue go out of scope, and crash as soon as you try using the `RooExtendPdf` instance.

Reply:
of course rvalues existed when RooFit was designed..? `RooExtendPdf(..., RooAbsPdf(...))`

> That's actually _desired_

Well then :smile:

Reply:
You have a point, though, that one could use rvalue refs, and take ownership.

Reply:
All I'm saying is: that used to compile (even at the time RooFit was designed), and now it doesn't. If it used to compile but was broken, that it's a _good thing_ that it does not compile anymore, so feel free to resolve the thread.